### ISSUE TITLE
CircleCI build broken

### ISSUE THREAD

**Issue by rgommers** [Quansight-Labs/quansight-labs-site#134]
See, e.g, gh-133. Despite `requirements.txt` containing the output of `pip freeze`, some dependency went missing in the last two weeks. `notebook==6.0.3` gets installed just fine according to the build log, but then the website build still ends in:
```
Scanning posts.....[2020-10-19T23:02:06Z] ERROR: Nikola: In order to build this site (compile ipynb), you must install the "notebook>=4.0.0" Python package.
[2020-10-19T23:02:06Z] ERROR: Nikola: Exiting due to missing dependencies.
```
That error doesn't mean `notebook` itself is missing, but some dependency of it (so import failed or some such reason, triggering a confusing error message).

The Netlify build, which uses the exact same `requirements.txt`, works just fine. 

If anyone wants to debug this, that'd be very welcome. I don't have time before the end of this week.

**Comment by melissawm** [comment]
I poked around a bit and got the site to build in the SSH debug session for CircleCI, but then it breaks for Netlify. The issue seems to be that CircleCI is using Python 3.9 and Netlify is on 3.7, so the dependencies clash. [This PR](https://github.com/Quansight-Labs/quansight-labs-site/pull/138) fixes CircleCI to run on 3.7 - let me know if that's reasonable.

**Comment by melissawm** [comment]
I think this has been solved by #138 and #139, so I'm closing - feel free to re-open if that's not the case.
